packages/components/src/Components/Inputs/FileUpload.tsx

```diff
@@ -42,14 +44,43 @@ interface FileUploadInputProps {
   };
 }
 
+interface FileRejection {
+  file: DpBlob;
+  errors: {
+    code: string;
+    message: string;
+  }[]
+}
+
 interface FileUploadInputState {
   files: DpBlob[];
+  pendingFiles: DpBlob[];
+  errorFiles: FileRejection[],
   progress: number;
   focused: boolean;
   hovered: boolean;
   error: string;
 }
 
+const FileError = ({
+  fileRejection,
+}: {
+  fileRejection: FileRejection;
+}) => {
+  const { file, errors } = fileRejection;
+  let message = errors[0].message;
+  if (errors[0].code === 'file-too-large') {
+    const maxSize = message.match(/(\d+) bytes/);
+    message = message.replace(maxSize[0], formatFileSize(parseInt(maxSize[1], 10)));
```

Comment:
That repo should not contain any phrases, I guess I should pass a phrase to translate to inject the size into instead of using the backend English one

Reply:
@JakeBiddell sorted, I was already passing the translated error message